doc/index.rst

```diff
@@ -22,6 +22,7 @@ Contents:
    reference
    auto_examples/index
    tutorials/index
+   auto_mayavi_examples/index
```

Comment:
but now this link will not exist unless mayavi is present... maybe it's just a sphinx warning so no big deal...